testcase_maker/executors/cpp.py

```diff
@@ -1,23 +1,26 @@
 from typing import TYPE_CHECKING, Union
 
 from testcase_maker.executor import Executor
+from testcase_maker.utils import run_command
 
 if TYPE_CHECKING:
     from pathlib import Path
 
 
 class CPPExecutor(Executor):
     """
-    !!! danger "Notice"
-            Not implemented yet.
+    Run your answer script in C++!
     """
 
     @property
     def file_extension(self) -> str:
         return "cpp"
 
     def compile(self, tempdir: Union["Path", str], source_filename: Union["Path", str]) -> Union["Path", str]:
-        pass
+        args = ["g++", "-o", str(source_filename.stem), str(source_filename)]
```

Comment:
```suggestion
        args = ["g++", "-o", str(source_filename.stem), str(source_filename.absolute())]
```

Tested, need to use absolute path here.